courses/machine_learning/intro_to_tensorflow/3_keras_sequential_api.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>If we set</p><p> </p><p>NUM_EVALS = 50</p><p>TRAIN_BATCH_SIZE = 1000</p><p> </p><p>then the rmse curves look much better (already with NUM_EVAL 10)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
ok thanks!